### PR TITLE
feat monitoring

### PR DESCRIPTION
**Pull request by petrjasek**
add monitoring view, it's a new page (like content) in the workspace,
showing same content like monitoring widget (former aggregate).

### CODE REVIEW

File: client/app/scripts/superdesk-desks/aggregate.js [hunk 1]
```diff
@@ -11,11 +11,12 @@
...
 
-    AggregateCtrl.$inject = ['api', 'session', 'desks', 'preferencesService', 'storage'];
-    function AggregateCtrl(api, session, desks, preferencesService, storage) {
+    AggregateCtrl.$inject = ['$scope', 'api', 'session', 'desks', 'preferencesService', 'storage', 'gettext'];
+    function AggregateCtrl($scope, api, session, desks, preferencesService, storage, gettext) {
         var PREFERENCES_KEY = 'agg:view';
         var defaultMaxItems = 10;
-        var self = this;
+        var self = this,
+            vm = this;
```
Comment (plamut) [inline client/app/scripts/superdesk-desks/aggregate.js]: IMO this should be unified, we don't really need two aliases to `this`.

File: client/app/scripts/superdesk-desks/aggregate.js [hunk 2]
```diff
@@ -52,9 +53,44 @@
...
                     this.groups = preference != null && preference.groups ? preference.groups : [];
                     this.loading = false;
+                    setupCards();
                 }));
         }));
 
+        /**
+         * Refresh view after setup
+         */
+        this.refresh = function() {
```
Comment (plamut) [inline client/app/scripts/superdesk-desks/aggregate.js]: ... and if we do have aliases for `this`, we should use them. :)

File: client/app/scripts/superdesk-desks/views/aggregate.html [hunk 3]
```diff
@@ -21,68 +21,12 @@
...
-                <div class="stage" ng-if="group.type === 'personal'" ng-show="total">
-                    <div class="stage-header clearfix">
-                        <div class="line"></div>
-                        <button class="toggle pull-left" ng-class="{closed: !agg.getState(group._id)}" ng-click="agg.switchState(group._id)">
-                            <i class="icon-chevron-down"></i>
-                        </button>
-                        <div class="stage-name pull-left" translate>Personal</div>
-                        <span class="label pull-right">{{total}}</span>
-                    </div>
+            <div class="desk" ng-repeat="group in agg.cards">
```
Comment (plamut) [inline client/app/scripts/superdesk-desks/views/aggregate.html]: `track by`?

File: client/app/scripts/superdesk-monitoring/monitoring.js [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+        var ITEM_HEIGHT = 57,
+            ITEMS_COUNT = 5,
+            BUFFER = 8,
+            UP = -1,
+            DOWN = 1,
+            ENTER_KEY = 13,
+            MOVES = {
+                38: UP,
+                40: DOWN
+            };
```
Comment (plamut) [inline client/app/scripts/superdesk-monitoring/monitoring.js]: Perhaps freeze this object (but not a big deal at all).

**Comment by akintolga** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

**Comment by amagdas** [comment]
@petrjasek  Going to merge this, please address the PR comments in a new PR.

**Comment by petrjasek** [comment]
:ok:

**Pull request merged**